docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx

```diff
@@ -22,7 +24,14 @@ import filter_contract from "/img/hubble/filter-contract.png";
 import persistent_only from "/img/hubble/persistent-only.png";
 import filtered_report from "/img/hubble/filtered-report.png";
 
-This page will guide you through an end-to-end analysis use case with Hubble and [Google Looker Studio](https://cloud.google.com/looker-studio?hl=en)
+This page will show how you can create visualizations with Hubble and [Google Looker Studio](https://cloud.google.com/looker-studio?hl=en).
+
+By the end of the tutorial you will have created two graphs that will help visualize temporary VS persistent contract data entry distribution and expiration.
+
+<img src={hubble_visualization_demo} width="1000" />
+<img src={hubble_visualization_final} width="500" />
+
+As you can see, persistent contract data entries account for roughly 25% of all contract data entries with the rest being temporary. There are also a lot more expired temporary contract data entries. This is expected because temporay entries cannot be bumped whereas persistent entries can be bumped/restored. Please see [State Archival - Contract Data Type Descriptions](https://developers.stellar.org/docs/learn/encyclopedia/storage/state-archival#contract-data-type-descriptions) for more information.
```

Comment:
nit: I would reword the last sentence: 
> For more information, you can read about [State Archival and the Contract Data Lifecycle].

Reply:
updated in c560544b37fe5785fb8eac095e6156ecc457b4e0